knowledge-base/grid-conditional-cell-background.md

```diff
@@ -23,197 +23,10 @@ res_type: kb
 
 ## Description
 
-I want to set the backcolor of the cells on a cell by cell basis. In some cases, setting the color for the entire row can also work.
+I want to customize the formatting of the cells on a cell by cell basis. In some cases, setting the color for the entire row can also work.
 
 ## Solution
 
-There are several approaches you can take depending on the goals and situation that you have:
-
-* You can use a particular column's [cell template]({%slug components/grid/features/templates%}#column-template) to render conditional markup inside its cell. In this case you will need some CSS to make your element take up the entire size of the cell so the default padding from the grid does not leave some of the original background visible. See **Example 1** below for this approach.
```

Comment:
I recommend keeping this content to provide options, but split it into proper headings and sections, with the conditional formatting through the events being at the top, and also offering an example of both the row and cell render event. You could render that snippet through a template in both the overview article and this kb to avoid code duplication